### PR TITLE
Bug 1587555: Add scopes necessary for docker-worker CI

### PR DESCRIPTION
**Pull request by walac**
```
vagrant@vagrant:community-tc-config  (docker-worker-scopes)$ TASKCLUSTER_ROOT_URL=https://community-tc.services.mozilla.com tc-admin diff --grep docker-worker
--- current
+++ generated
@@ -3,9 +3,22 @@
   - Role=repo:.*
   - Role=project-admin:.*
   - Role=github-org-admin:.*
   - Role=github-team:.*
   - Role=login-identity:.*
   - WorkerPool=proj-.*

 resources:
+  Role=repo:github.com/taskcluster/docker-worker:
+    roleId: repo:github.com/taskcluster/docker-worker
+    description:
+      *DO NOT EDIT* - This resource is configured automatically.

+
+    scopes:
+      - docker-worker:cache:docker-worker-garbage-*
+      - docker-worker:capability:device:loopbackAudio
+      - docker-worker:capability:device:loopbackVideo
+      - docker-worker:capability:privileged
+      - queue:create-task:highest:proj-taskcluster/ci
+      - secrets:get:repo:github.com/taskcluster/docker-worker/ci-creds
+      - secrets:get:repo:github.com/taskcluster/docker-worker/pulse-creds
```

### CODE REVIEW

File: generate/repo_access.py [hunk 1]
```diff
@@ -34,6 +34,22 @@
...
+            'docker-worker:capability:device:loopbackAudio',
+            'docker-worker:capability:device:loopbackVideo',
+            'secrets:get:repo:github.com/taskcluster/docker-worker/ci-creds',
+            'secrets:get:repo:github.com/taskcluster/docker-worker/pulse-creds',
+        ],
+        'worker-pools': [
+            'proj-taskcluster/ci',
+        ],
+        'for': [
+            'github.com/taskcluster/docker-worker',
```
Comment (djmitche) [inline generate/repo_access.py]: You'll want `:*` at the end of this.

File: generate/repo_access.py [hunk 2]
```diff
@@ -34,6 +34,22 @@
             'github.com/taskcluster/taskcluster:*',
         ]
     },
+    {
+        'scopes': [
+            'docker-worker:cache:docker-worker-garbage-*',
+            'docker-worker:capability:privileged',
+            'docker-worker:capability:device:loopbackAudio',
+            'docker-worker:capability:device:loopbackVideo',
```
Comment (djmitche) [inline generate/repo_access.py]: These make sense here..

File: generate/repo_access.py [hunk 3]
```diff
@@ -34,6 +34,22 @@
...
+            'secrets:get:repo:github.com/taskcluster/docker-worker/ci-creds',
+            'secrets:get:repo:github.com/taskcluster/docker-worker/pulse-creds',
```
Comment (djmitche) [inline generate/repo_access.py]: ..but for these I'd rather use secrets under `project/taskcluster/`, e.g., `project/taskcluster/docker-worker/{ci,pulse}-creds`.

File: generate/repo_access.py (hunk 1 above)
Comment (walac) [reply generate/repo_access.py]: Done

File: generate/repo_access.py (hunk 3 above)
Comment (walac) [reply generate/repo_access.py]: Done

File: generate/repo_access.py [hunk 4]
```diff
@@ -34,6 +34,22 @@
...
+            'docker-worker:capability:device:loopbackAudio',
+            'docker-worker:capability:device:loopbackVideo',
+            'project/taskcluster/docker-worker/ci-creds',
+            'project/taskcluster/docker-worker/pulse-creds',
+        ],
+        'worker-pools': [
+            'proj-taskcluster/ci',
+        ],
+        'for': [
+            'github.com/taskcluster/docker-worker*',
```
Comment (djmitche) [inline generate/repo_access.py]: A `:` here will prevent other repos beginning with `docker-worker` from matching.

File: generate/repo_access.py [hunk 5]
```diff
@@ -34,6 +34,22 @@
...
         ]
     },
+    {
+        'scopes': [
+            'docker-worker:cache:docker-worker-garbage-*',
+            'docker-worker:capability:privileged',
+            'docker-worker:capability:device:loopbackAudio',
+            'docker-worker:capability:device:loopbackVideo',
+            'project/taskcluster/docker-worker/ci-creds',
+            'project/taskcluster/docker-worker/pulse-creds',
```
Comment (djmitche) [inline generate/repo_access.py]: `.../testing/...`

Comment (walac) [reply generate/repo_access.py]: `project/taskcluster/docker-worker/testing/`?

Comment (djmitche) [reply generate/repo_access.py]: I'm trying to gather all of the testing-related secrets under `project/taskcluster/testing/..`:
https://community-tc.services.mozilla.com/secrets

**Pull request merged**